tools/wake-format/emitter.cpp

```diff
@@ -951,7 +951,8 @@ wcl::doc Emitter::walk_data(ctx_t ctx, CSTElement node) {
   MEMO(ctx, node);
   FMT_ASSERT(node.id() == CST_DATA, node, "Expected CST_DATA");
 
-  auto fmt_members = fmt().walk_all(fmt().walk(WALK_NODE).freshline().consume_wsnlc());
+  auto fmt_members = fmt().walk(WALK_NODE).consume_wsnlc().walk_all(
```

Comment:
Ok so as I understand it, what you're doing here is assuming that there will always be at least one node (seems correct), and emitting one less freshline by emitting a leading freshline on everything else instead of a trailing newline. Got it.

Reply:
Yep, exactly. That is generally the policy (there are other cases of this) when there are extra "separators" at the end of a sequence of things.